### PR TITLE
3-GET/api/topics task

### PR DESCRIPTION
**Pull request by Alhariri97**
Setting up and creating  the module, controller, test files.
Start the first rout GET/api/topics with tests.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("status:200, responds with an Array has 3 objects topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
```
Comment (dappernerddesigns) [inline __tests__/app.test.js]: We are expecting the response to be an object, with a key and a value of an array. Have a look at your endpoints.json for an idea of what I mean.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+
+const { getTopics } = require("./controllers/topics.controller");
+
+const app = express();
+
+app.use(express.json());
```
Comment (dappernerddesigns) [inline app.js]: Do you need this for get requests?

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
Some minor tweaks. If you could take a look and DM me when done so I can approve it for you :) V

File: app.js (hunk 2 above)
Comment (Alhariri97) [reply app.js]: I don't need them for the get request , but I thought  I'll need json use when posting to our data so I just wrote them .

**Review by dappernerddesigns — APPROVED** [review]
Fantastic, merge when ready :)

**Pull request merged**